src/main/java/com/google/firebase/database/FirebaseDatabase.java

```diff
@@ -45,6 +46,8 @@
  */
 public class FirebaseDatabase {
 
+  static final String FIREBASE_RTDB_EMULATOR_HOST_ENV_VAR = "FIREBASE_RTDB_EMULATOR_HOST";
+  static final String RTDB_EMULATOR_HOST = "localhost:9000";
```

Comment:
I think the env variable should contain the emulator host URL -- not just a `true` value.

@schmidt-sebastian can you also take a look?

Reply:
@sarajmunjal is working on it. Please consider this PR WIP until we remove the WIP tag and re-request review.

Reply:
Done